addons/l10n_ar/views/report_invoice.xml

```diff
@@ -4,19 +4,19 @@
     <!-- this header can be used on any Argentinean report, to be useful some variables should be passed -->
     <template id="custom_header">
         <div>
-            <div class="row" t-if="not pre_printed_report">
-                <div name="left-upper-side" class="col-5">
+            <div class="row">
+                <div name="left-upper-side" class="col-5" t-if="not pre_printed_report">
                     <img t-if="o.company_id.logo" t-att-src="image_data_uri(o.company_id.logo)" style="max-height: 45px;" alt="Logo"/>
                 </div>
                 <div name="center-upper" class="col-2 text-center" t-att-style="'color: %s;' % o.company_id.primary_color">
                     <span style="display: inline-block; text-align: center; line-height: 8px;">
                         <h1 style="line-height: 25px;">
-                            <strong><span t-esc="document_letter"/></strong>
+                            <strong><span t-esc="not pre_printed_report and document_letter or '&#160;'"/></strong>
```

Comment:
@Bruno-Zanotti why the #160?

Reply:
It's a non-breaking space character for html to generate the necessary space avoiding to put something like:
```
<t t-if="not pre_printed_report">
    <strong><span t-esc="document_letter"/></strong>
</t> 
<t t-else=""> 
    <br/> 
</t> 
```

It is also used in other places in Odoo, eg: [link](https://github.com/odoo/odoo/blob/65b0338883e173c792ee177eb438d411f214bb8d/addons/partner_autocomplete/static/src/xml/partner_autocomplete.xml#L10)

Reply:
Why not put nbsp then?

Reply:
@jco-odoo 
I've been testing it and the nbsp doesn't work with the t-esc tag. It could work instead of <br/> in the t-else of the second possibility.

Reply:
@Gorash I was told by @Bruno-Zanotti that t-esc="'&#nbsp;'" does not work.  I wonder if &amp;nbsp; would have worked... But I would suppose the t-esc immediately translates into the final xml.

Reply:
@jco-odoo `&nbsp;` is not an xml default entities (ref: lxml.etree.parse), is't a html entities. The `&amp;nbsp;` is used when it's use double decode.